grr/server/grr_response_server/gui/api_plugins/timeline.py

```diff
@@ -111,18 +111,28 @@ def Handle(
       message = f"Hunt '{hunt_id}' is not a timeline hunt"
       raise ValueError(message)
 
+    if (args.format != ApiGetCollectedTimelineArgs.Format.RAW_GZCHUNKED) and (
+        args.format != ApiGetCollectedTimelineArgs.Format.BODY):
```

Comment:
No need for the internal parens, `!=` binds stronger than `and`. External ones are needed because of the linebreak, though.

Reply:
Thank you! I updated it